docs/synapse/userguides/storm_ref_filter.rstorm

```diff
@@ -793,247 +917,449 @@ know the exact order or values of the array itself.
 ::
-    
-    <query> +#rep.eset.seduploader
+  
+  <query> -#*.*.seduploader
+
+The Vertex Project uses the ``cno`` root tag to represent our own internal assessments (and distinguish them from
+third party-assessments), and the ``mal`` element to represent assessments related to malware. The tag glob filter
+above uses two single asterisks to match any element in both the first and second positions, and will match all of
+the following:
+
+- ``rep.eset.seduploader``
+- ``rep.paloalto.seduploader``
+- ``cno.mal.seduploader``
 
+...etc.
 
-- Filter results to exclude nodes tagged as being associated with the TOR network:
 
+*Filter the current working set to include any nodes reported by Microsoft whose tags end in "blizzard":*
 
-.. storm-pre:: inet:ipv4 -#cno.infra.anon.tor
+.. storm-pre:: [ ( inet:fqdn=lotorgas.ru +#rep.microsoft.aqua_blizzard ) ( inet:fqdn=justiceua.org +#rep.microsoft.cadet_blizzard ) ( inet:fqdn=actblues.com +#rep.microsoft.forest_blizzard ) ( inet:fqdn=scandefinform.com +#rep.microsoft.star_blizzard ) ] +#rep.microsoft.**blizzard
 ::
-    
-    <query> -#cno.infra.anon.tor
+  
+  <query> +#rep.microsoft.**blizzard
 
+The tag glob filter above uses a double asterisk to match any Microsoft tag (tag that begins ``rep.microsoft``)
+that ends in "blizzard", regardless of tag depth. The filter will match all of the following:
 
-- Filter results to exclude nodes tagged as sinkholes:
+- ``rep.microsoft.aqua_blizzard``
+- ``rep.microsoft.cadet_blizzard``
+- ``rep.microsoft.forest_blizzard``
+- ``rep.microsoft.very.long.tag.thatendswithblizzard``
 
+...etc.
 
-.. storm-pre:: inet:ipv4 -#cno.infra.dns.sinkhole
+*Filter the current working set to exclude any nodes tagged with any tag that starts with "cno" and is followed by any string:*
+
+.. storm-pre:: [ ( inet:ipv4=93.90.223.185 +#cno.infra.dns.sink.hole ) ( inet:fqdn=vertex.link +#cno ) ] -#cno**
 ::
-    
-    <query> -#cno.infra.dns.sinkhole
+  
+  <query> -#cno**
+
+The tag glob filter above uses a double asterisk to match any string following "cno". The filter will match all of
+the following:
 
+- ``cno.mal``
+- ``cno.threat.t42``
+- ``cnoooo.you_get_a_cno.and_you_get_a_cno``
 
-*Filter by tag and time:*
+...etc.
 
-- Filter results to include only nodes that were associated with TOR infrastructure as of December 12, 2019:
+.. NOTE::
+  
+  The double asterisk must match "something" - the filter above will **not** match a node that simply has the tag ``#cno``.
 
 
-.. storm-pre:: inet:fqdn +#cno.infra.anon.tor@=2019/12/12
+*Filter the current working set to include any nodes tagged with any "APT" group by any third-party reporting organization:*
+
+.. storm-pre:: [ ( inet:fqdn=www.autoapec.com +#rep.alienvault.apt30 ) ( inet:fqdn=intuitstatistic.com +#rep.mandiant.apt28 ) ( inet:fqdn=supportus.mefound.com +#rep.pwc.apt10 ) ] +#rep.*.apt**
 ::
-    
-    <query> +#cno.infra.anon.tor@=2019/12/12
+  
+  <query> +#rep.*.apt**
```

Comment:
this actually only needs to be a single asterisk at the end for the example matches

Reply:
Derp. Changed example to show better use of double asterisks.